pkgs/by-name/de/decent-sampler/package.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, fetchzip
+, buildFHSEnv
+, alsa-lib
+, freetype
+, nghttp2
+, xorg
+, }:
+
+let
+  pname = "decent-sampler";
+  version = "1.9.4";
+
+  decent-sampler = stdenv.mkDerivation {
+    inherit pname version;
+
+    src = fetchzip {
+      # dropbox link: https://www.dropbox.com/sh/dwyry6xpy5uut07/AABBJ84bjTTSQWzXGG5TOQpfa\
+
+      url = "https://archive.org/download/decent-sampler-linux-static-download-mirror/Decent_Sampler-${version}-Linux-Static-x86_64.tar.gz";
+      sha256 = "sha256-O/0R70tZOmSGgth6nzt4zPiJr1P8890uzk8PiQGnC6M=";
```

Comment:
```suggestion
      hash = "sha256-O/0R70tZOmSGgth6nzt4zPiJr1P8890uzk8PiQGnC6M=";
```

Reply:
borgo also identifies a hash mismatch here:

```
error: hash mismatch in fixed-output derivation '/nix/store/vf1z2cly6kzd9par02zfqbral6nf8rw4-source.drv':
         specified: sha256-O/0R70tZOmSGgth6nzt4zPiJr1P8890uzk8PiQGnC6M=
            got:    sha256-lTp/mukCwLNyeTcBT68eqa7aD0o11Bylbd93A5VCILU=
```

Reply:
I encountered this too.

Reply:
thanks guys, I just realized that I built it for version 1.9.8 first from dropbox, but then changed to archive.org for better reliability but forgot to reset the hash.